metadata.c

```diff
@@ -422,6 +422,29 @@ int pv_metadata_rm_usermeta(const char *key)
 	return -1;
 }
 
+static void pv_unescape_data(char *buf, int size)
+{
+	const char chars[] = {'\"', '\\'};
+	int i = -1;
+	int j = 0;
+
+	while (i < size) {
+next:
+		++i;
+		if (buf[i] == '\\') {
+			for (size_t k = 0; k < sizeof(chars); ++k) {
+				if (buf[i + 1] == chars[k])
+					goto next;
```

Comment:
better use continue here, right?

Reply:
Noy really, because I need to break the inner loop and jump to the beginning of the outer loop, So the alternative is mixing `break` with a flag to avoid the execution of the last part of the outer loop